config/formats.js

```diff
@@ -89,7 +89,7 @@ exports.Formats = [
 		banlist: ['RU', 'BL3', 'Drought'],
 	},
 	{
-		name: "[Gen 7] PU (alpha)",
+		name: "[Gen 7] PU (beta)",
 
 		mod: 'gen7',
 		ruleset: ['[Gen 7] NU (beta)'],
```

Comment:
as far as I was concerned it was, so I don't think it's ok in that case. Anything else that needs to be done?